src/ToggleButton.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts" context="module">
+  import { capitalize } from "@app/utils";
```

Comment:
Out of curiosity, is there some benefit to import `capitalize` in the module context?

Reply:
Good catch! 6d2574acb69f6723d33d65620d0187c463099579